spec/cypress/integration/CipLessons.feature

```diff
@@ -2,7 +2,7 @@ Feature: Core Induction Programme lessons
   Users should be able to view and sometimes edit cip lessons.
 
   Scenario: Admins can edit lessons
-    Given I am logged in as "admin"
+    Given I am admin and logged in
```

Comment:
I hadn't seen how the Given statements were made. I half assumed it was Percy wizadry but is it matching the "I am admin and logged in" and then running the function loginAdmin() written in Cypress? So Given takes a string and function as its parameters?

Reply:
Let's chat about it in person on Tuesday :)

The short gist of it is that we define these steps in `spec/cypress/support/step_definitions/factory-bot.js`.